### PR TITLE
layouts

### PR DESCRIPTION
**Pull request by imubarak234**
- Delete all the boilerplate from CRA (text, images, styles).

- Create a directory called components.

- Inside components, create a new Calculator.js file.

- In Calculator.js, create a React component that matches this design:

- Don't add any extra design.

- Develop just the UI of the calculator; the math logic to make it work will be added in the next project.

- Use class-based components.

- Once the Calculator is ready, import it and use it in your main component (App.js).

- Test it in the browser; it should render without problems.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
+// import Calculator from './components/Calculator';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
+// import App from './App';
+// import reportWebVitals from './reportWebVitals';
```
Comment (DeliceLydia) [inline src/index.js]: - [ ] Please remove commented codes in this file

File: src/App.js [hunk 2]
```diff
@@ -1,29 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+// import logo from './logo.svg';
+// import './App.css';
```
Comment (DeliceLydia) [inline src/App.js]: - [ ] Remove all the commented codes

Comment (DeliceLydia) [reply src/App.js]: - [ ] In the `src` folder I suggest that you keep files you are using only you don't need like, you can keep only `css` file you are using only
ff

- [ ] In the `public` folder you can just keep `index.html` only that is what you want for now

File: src/App.js [hunk 3]
```diff
@@ -1,29 +1,9 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+  return <Calculator />;
```
Comment (DeliceLydia) [inline src/App.js]: - [ ] `app` component should be class-based please fix that

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @imubarak234, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Here are major highlights for your project:
- No linter error
- Design look okey


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omar-labana — APPROVED** [review]
Hi @imubarak234, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**